packages/eslint-plugin/src/configs/react.js

```diff
@@ -79,8 +79,7 @@ const config = {
         ignorePatterns: [
           'require(<.*?>)?\\(',
           'https?:\\/\\/',
-          '^(import|export) \\{ \\w+( as \\w+)? \\} from',
-          '^import \\* as',
+          '^(import|export)',
```

Comment:
Probably good to make sure it has a space after import/export
```suggestion
          '^(import|export) ',
```